src/index.ts

```diff
@@ -86,6 +81,7 @@ const defaultLoaders = Object.freeze({
   noExt: loaders.loadYaml,
 } as const);
 const defaultLoadersSync = Object.freeze({
+  '.mjs': loaders.loadJsSync,
```

Comment:
I don't think that works. ESM can only load async.

Reply:
Yep, made a mistake when solving conflicts. Updated now